src/main/java/io/jenkins/plugins/orka/client/OrkaVMConfig.java

```diff
@@ -0,0 +1,73 @@
+package io.jenkins.plugins.orka.client;
+
+import com.google.gson.annotations.SerializedName;
+
+public class OrkaVMConfig {
+
+    @SerializedName("orka_vm_name")
+    private String name;
+
+    @SerializedName("orka_cpu_core")
+    private int cpuCount;
+
+    @SerializedName("orka_base_image")
+    private String baseImage;
+
+    public OrkaVMConfig(String name, String deploymentStatus, int cpuCount, String baseImage, String image,
+            String configurationTemplate) {
+        this.name = name;
+        this.cpuCount = cpuCount;
+        this.baseImage = baseImage;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public int getCPUCount() {
+        return this.cpuCount;
+    }
+
+    public String getBaseImage() {
+        return this.baseImage;
+    }
+
+    @Override
+    public int hashCode() {
+        final int prime = 31;
+        int result = 1;
+        result = prime * result + ((baseImage == null) ? 0 : baseImage.hashCode());
+        result = prime * result + cpuCount;
+        result = prime * result + ((name == null) ? 0 : name.hashCode());
+        return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj) {
+            return true;
+        }
+        if (obj == null) {
+            return false;
+        }
+        if (getClass() != obj.getClass()) {
+            return false;
+        }
+        OrkaVMConfig other = (OrkaVMConfig) obj;
+        if (baseImage == null) {
+            if (other.baseImage != null) {
```

Comment:
nit: Can we use `if (baseImage == null && other.baseImage != null) { return false; }` instead of nested `if`s?

Reply:
That's autogenerated. I prefer not to touch it